src/components/Frame/Frame.tsx

```diff
@@ -363,6 +363,14 @@ class Frame extends React.PureComponent<CombinedProps, State> {
     this.setState({skipFocused: false});
   };
 
+  private handleClick = () => {
+    if (this.mainContentNode.current == null) {
+      return;
+    }
+
+    this.mainContentNode.current.focus();
```

Comment:
main isn't a focusable element

Reply:
The interesting thing is that this works as expected as it ends up moving the focus to the correct location. It looks like chrome moves the focus to the div, therefore, tabbing works great. Safari ends up moving the focus to the heading which is really nice.

@dpersing this came to light when writing tests. The `active element` will never be the `main` element unless we actually add a `tabIndex` to it. Do you see an issue with adding a tabIndex to `main`?

Reply:
> Do you see an issue with adding a tabIndex to main?

That should be fine! It's how focus is managed to modals currently.